packages/channel-provider/src/messaging-service.ts

```diff
@@ -109,8 +109,6 @@ export class MessagingService {
         } else if (isJsonRpcErrorResponse(event.data)) {
           reject(new RpcError(event.data.error));
         }
-      } else {
-        logger.error({event}, 'Received a non JsonRpc response');
```

Comment:
I am in favor of somehow noting/logging that the event cannot be interpreted so that we don't silently drop events. Maybe not possible given the infinite loop problem at the moment.

Reply:
https://github.com/statechannels/monorepo/pull/2138#discussion_r438973814